composer.json

```diff
@@ -59,7 +59,11 @@
         "symfony/http-kernel": "^3.0",
         "symfony/intl": "^3.0",
         "symfony/options-resolver": "^3.0",
-        "symfony/polyfill": "^1.6",
+        "symfony/polyfill-apcu": "~1.1",
+        "symfony/polyfill-intl-icu": "~1.0",
+        "symfony/polyfill-mbstring": "~1.0",
+        "symfony/polyfill-php56": "~1.0",
+        "symfony/polyfill-php70": "~1.0",
```

Comment:
same as above

Reply:
No, I just copied the default polyfills from a new Symfony 3.3 project. I don't need them & I can drop them if you want.

Reply:
@wimvds ok you can drop them and only add mbstring and xml as polyfill.